website/docs/sql-reference/clauses/sql-order-by.md

```diff
@@ -0,0 +1,65 @@
+---
+id: order-by
+title: SQL ORDER BY
+description: The ORDER BY clause allows you to specify the resulting row order for a query.
+slug: /sql-reference/order-by
+---
+
+<head>
+    <title>Working with the SQL ORDER BY clause</title>
+</head>
+
+The ORDER BY clause allows you to specify the resulting row order for a query. In practice, you use the ORDER BY clause to indicate which field(s) you want to order by and in what type of order you want (ascending or descending). It’s useful to leverage during ad hoc analyses and for creating appropriate column values for partitioned rows in window functions.
```

Comment:
May want to break out the last sentence here into its own paragraph (kind of bled together for me in reading).

Reply:
Could also shorten to "in window functions" (and elaborate later), was a little earlier on the page to grasp that partitioning piece.